client/me/purchases/purchases-site/reconnect-notice.jsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import { localize } from 'i18n-calypso';
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Notice from 'components/notice';
+import NoticeAction from 'components/notice/notice-action';
+
+class PurchaseReconnectNotice extends Component {
+	static propTypes = {
+		name: React.PropTypes.string,
+		domain: React.PropTypes.string,
+	}
+
+	render() {
+		const { translate, name, domain } = this.props;
+
+		return (
+			<Notice
+				showDismiss={ false }
+				status="is-error"
+				text={ translate( '%(site)s has been disconnected from WordPress.com.', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( '%s has been disconnected.' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).